### DISCUSSION TITLE
paginator event function

### DISCUSSION THREAD

**Discussion by theoreaves** [livewire/livewire#5522] [1. Help]
Is there a lifecycle hook for the paginator like their is for wire models. 

So, like you can add a "updated" function to capture when a field changes, is there something like that for the paginator?

I found gotoPage, but it seems to stop the standard pagination from happening.  I still want the page change to happen like normal, I just want to be aware so I can react to it.

**Comment by joshhanley** [comment]
@theoreaves yep, should just be `updatingPage()/ updatedPage()` (assuming your paginator is using the "page" default).

It runs if next/previous/set/etc get called.

It gets triggered here:

https://github.com/livewire/livewire/blob/e92233f31dc1b88fb1ac242b32afe1ab7525ee05/src/WithPagination.php#L87-L105

Hope this helps!
